tests/stub/summary/test_summary.py

```diff
@@ -247,6 +235,793 @@ def test_multiple_notifications(self):
+    def test_success(self):
+        summary = self._get_summary("empty_summary_type_r.script")
+        self.assertEqual(len(summary.gql_status_objects), 1)
+        self.assert_is_success(summary.gql_status_objects[0])
+
+    def test_omitted_result(self):
+        summary = self._get_summary(
+            "empty_summary_type_r_omitted_result.script"
+        )
+        self.assertEqual(len(summary.gql_status_objects), 1)
+        self.assert_is_omitted_result(summary.gql_status_objects[0])
+
+    def test_no_data(self):
+        summary = self._get_summary(
+            "empty_summary_type_r_no_data.script"
+        )
+        self.assertEqual(len(summary.gql_status_objects), 1)
+        self.assert_is_no_data(summary.gql_status_objects[0])
+
+    def test_multiple_statuses(self):
+        # order is not GQL compliant, but the driver should not touch it
+        in_statuses = [
+            self.make_test_status(
+                "01N01", "warning",
+                subcondition="test subcondition. Don't do this™.", i=1,
+            ),
+            SUCCESS_GQL_STATUS_OBJECT,
+            NO_DATA_GQL_STATUS_OBJECT,
+            self.make_test_status(
+                "03N01", "informational",
+                subcondition="test subcondition. Do that™.", i=2,
+            ),
+            self.make_test_status(
+                "01N00", "warning",
+                subcondition="test subcondition", i=3,
+            ),
+            self.make_test_status(
+                "01N00", "warning",
+                subcondition="test subcondition",
+                classification="FOOBAR", severity=None, i=4,
+            ),
+            OMITTED_GQL_STATUS_OBJECT,
+            self.make_test_status(
+                "03N03", "informational",
+                subcondition="test subcondition. Here we go again.™.", i=5,
+            ),
+        ]
+
+        summary = self._get_summary(
+            "summary_with_statuses.script",
+            vars_={"#STATUSES#": json.dumps(in_statuses)}
+        )
+
+        self.assertEqual(len(summary.gql_status_objects), 8)
+        self.assert_is_test_gql_status_object(
+            summary.gql_status_objects[0], "01N01", "warning",
+            subcondition="test subcondition. Don't do this™.", i=1,
+        )
+        self.assert_is_success(summary.gql_status_objects[1])
+        self.assert_is_no_data(summary.gql_status_objects[2])
+        self.assert_is_test_gql_status_object(
+            summary.gql_status_objects[3], "03N01", "informational",
+            subcondition="test subcondition. Do that™.", i=2,
+        )
+        self.assert_is_test_gql_status_object(
+            summary.gql_status_objects[4], "01N00", "warning",
+            subcondition="test subcondition", i=3,
+        )
+        self.assert_is_test_gql_status_object(
+            summary.gql_status_objects[5], "01N00", "warning",
+            subcondition="test subcondition",
+            raw_classification="FOOBAR", classification="UNKNOWN",
```

Comment:
```suggestion
            raw_classification="FOOBAR", classification=None,
```
Java driver does not have the `UNKNOWN` parsed value, it returns an empty `Optional` is such cases. The backend can simulate `UNKNOWN` value for Testkit, but it lead to some other tests failure.

Reply:
see https://github.com/neo4j-drivers/testkit/pull/617#discussion_r1634439087